### PR TITLE
Convert PS-OCT MAT files to OME-Zarr

### PR DESCRIPTION
**Pull request by calvinchai**

### CODE REVIEW

**Comment by calvinchai** [comment]
Add a note here: I have test for OCT pipeline locally but I did not upload it since the test files are too big. We will figure out what to do when we are more close to the end.

**Comment by calvinchai** [comment]
Refactor the pyramid part is done. More need to be refactored will be done when I work on other modalities. Now it also lazy loads the files instead of loading all at once.

File: linc_convert/modalities/psoct/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""Dark Field microscopy converters."""
```
Comment (kabilar) [inline linc_convert/modalities/psoct/__init__.py:1]:
```suggestion
"""Polarization-sensitive optical coherence tomography converters."""
```

File: linc_convert/modalities/psoct/cli.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+"""Entry-points for Dark Field microscopy converter."""
```
Comment (kabilar) [inline linc_convert/modalities/psoct/cli.py:1]:
```suggestion
"""Entry-points for polarization-sensitive optical coherence tomography converter."""
```

**Comment by balbasty** [comment]
I went (very very) quickly thourgh it and it looks fine. 

The only question I have, and I think we'll have to check with Nate, is how the channels of the input file map to its spatial axes. Say differently, each voxel contains an orientation vector and we need to understnad in what referential this orientation vector is. In the output zarr, we'll want the first channel to correspond to dimension "x", the second channel to dimension "y", etc.

**Comment by calvinchai** [comment]
Probably we can revisit that later? We can open an issue and merge this first. 

> I went (very very) quickly thourgh it and it looks fine.
> 
> The only question I have, and I think we'll have to check with Nate, is how the channels of the input file map to its spatial axes. Say differently, each voxel contains an orientation vector and we need to understnad in what referential this orientation vector is. In the output zarr, we'll want the first channel to correspond to dimension "x", the second channel to dimension "y", etc.

**Pull request merged**